tracer/src/Datadog.Trace/AppSec/Security.cs

```diff
@@ -347,6 +350,60 @@ void SetHtmlResponseContent()
             return blockingAction;
         }
 
+        private string GetJsonResponse()
+        {
+            if (_blockedJsonTemplateCache != null)
+            {
+                return _blockedJsonTemplateCache;
+            }
+
+            if (_settings.BlockedJsonTemplatePath != null && AppDomain.CurrentDomain.BaseDirectory != null)
+            {
+                var fullPath =
+                    Path.IsPathRooted(_settings.BlockedJsonTemplatePath)
```

Comment:
NIT: Path.Combine() already checks if the second argument is rooted, so this expression could be simplified

Reply:
I thought that was probably the case, but I was sure, I think it's probably a good thing to have the desired behaviour a bit more explicit.